docs/rounding.md

```diff
@@ -0,0 +1,79 @@
+## Rounding a ZonedDateTime
+
+(Only supported in Julia version 0.5 and later.)
+
+Rounding operations (`floor`, `ceil`, and `round`) on `ZonedDateTime`s are performed in a
+[similar manner to `DateTime`](http://julia.readthedocs.org/en/latest/manual/dates/#rounding)
+and should generally behave as expected. When `VariableTimeZone` transitions are involved,
+however, unexpected behaviour may be encountered.
+
+Instead of performing rounding operations on the `ZonedDateTime`'s internal UTC `DateTime`,
```

Comment:
That's totally fair. I considered that when I was writing it, but I figured that some people who do know about the internals might wonder why we're not just using UTC, so I wanted to put this explanation *somewhere*.

Do you think this should be moved to a comment in the code, perhaps? Or just removed entirely?